### PR TITLE
Fully translated CommonEvents.json

### PR DESCRIPTION
**Pull request by BladeSides**

### CODE REVIEW

File: en/CommonEvents.json [hunk 1]
```diff
@@ -42,114 +42,114 @@
...
+        "Does any of this make sense?",
+        "Are you fighting me?",
+        "\\!Go on."
     ],
     "604": [
         [
-            "책장에 책을 끼운다",
-            "아무것도 하지 않는다"
+            "I put a book in the shelf,",
+            "but it did nothing."
```
Comment (Creta5164) [inline en/CommonEvents.json:120]: This is selection dialogue that means 'Put a book in the shelf' and 'Do noting'.

File: en/CommonEvents.json [hunk 2]
```diff
@@ -42,114 +42,114 @@
...
-        "....하. 이 선이 또 빠진 거였어.",
-        "\\!어떤 미친놈이 느리다고 또 발로 찼나보네.",
-        "자, 이제 준비 다 끝났으니까 직접 방송 해봐.",
-        "\\!컴퓨터에 있는 마이크 버튼 누르면서",
-        "거기 써져있는 대본대로 읊으면 돼."
+        "Handed the driver.",
+        "Did you bring the driver?",
+        "\\!Let's see.",
+        "....Heh. Missing line again.",
+        "\\!Some guy kicked cause he was slow.",
```
Comment (Creta5164) [inline en/CommonEvents.json:145]: 미친놈 is means like 'crazy person' like thing.

File: en/CommonEvents.json [hunk 3]
```diff
@@ -42,114 +42,114 @@
...
+        "\\!Press the microphone button and",
+        "read the script written there."
     ],
     "607": [
-        "열쇠를 사용하여 배전함의 잠금을 풀었다.",
-        "...그러나 아직 경첩에 번호 자물쇠가 남아 있다.",
-        "보건실 열쇠를 획득했다."
+        "The distribution box was unlocked.",
+        "...There's a number lock on the hinge.",
+        "Obtained the health room key."
```
Comment (Creta5164) [inline en/CommonEvents.json:153]: 그러나 is does like 'but', is there reason to wrote like this?

File: en/CommonEvents.json [hunk 4]
```diff
@@ -22,24 +22,24 @@
...
-        "검은 손은 앞을 보지 못하는 대신",
-        "\\C[1]청각\\C[0]이 뛰어나.",
-        "물론, \\!그 놈이 바로 앞에 있어도",
-        "조용히 걸으면서 지나가면 별 탈 없을 거야.",
-        "쫓기는 중이라면 적당히 거리가 벌어졌을 때",
-        "천천히 걸어서, 그 장소를 벗어나도 되고."
+        "There was a loud noise in the hallway.",
+        "\\!What happened?",
+        "You were chased by a big hand guy?",
+        "There's a \\C[2]black hand\\C[0]guy.",
```
Comment (Creta5164) [inline en/CommonEvents.json:28]: Use 'ghost' instead 'guy' or use more effective word if you have suggest us.

File: en/CommonEvents.json [hunk 5]
```diff
@@ -22,24 +22,24 @@
...
+        "distance to make your escape."
     ],
     "500": [
         " "
     ],
     "510": [
-        "뭐 궁금한 거 있나?\\!",
-        "내가 아는 선에선 최대한 알려주지."
+        "Do you have any questions?\\!",
+        "I'll tell you what I know!"
```
Comment (Creta5164) [inline en/CommonEvents.json:42]: Change ! to . because this word is spoken by character that serious person.

File: en/CommonEvents.json [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
+        "They're annoying.",
+        "If you encounter it, get out of sight.",
+        "\\!...That's all.",
+        "\\!After knowing, they're less dangerous.",
+        "How are you going to find the students?",
+        "\\!Suddenly got dark?",
+        "I've been through a lot so far.",
+        "\\!There's so many types.",
+        "It's hard to say it...",
+        "I call it the \\C[1]ghost phenomenon\\C[0].",
```
Comment (Creta5164) [inline en/CommonEvents.json:19]: 폴터가이스트 is means Poltergeist, use this word instead ghost phenomenon after of it.

File: en/CommonEvents.json [hunk 7]
```diff
@@ -1,27 +1,27 @@
...
+        "\\!The finest people disappear.",
+        "I'm doing a trick.",
+        "\\C[2]With a strange sound",
+        "If a nonsense phenomenon\\C[0] appears",
+        "\\!Just try to get out of there.",
```
Comment (Creta5164) [inline en/CommonEvents.json:24]: This paragraph is also separated.  
It's looks like this :  
>이상한 소리가 들리면서 말도 안 되는 현상이 나타난다면 그냥 그곳을 최대한 벗어나려고 해봐.

File: en/CommonEvents.json [hunk 8]
```diff
@@ -1,27 +1,27 @@
...
+        "\\!...That's all.",
+        "\\!After knowing, they're less dangerous.",
+        "How are you going to find the students?",
+        "\\!Suddenly got dark?",
+        "I've been through a lot so far.",
+        "\\!There's so many types.",
+        "It's hard to say it...",
+        "I call it the \\C[1]ghost phenomenon\\C[0].",
+        "\\!The finest people disappear.",
+        "I'm doing a trick.",
```
Comment (Creta5164) [inline en/CommonEvents.json:21]: This paragraph is separated, it's looks like this :
> 잔챙이 놈들이 모습을 감추고 수작질을 부리는 거지.

잔챙이 is means 'small fry' according [here](https://en.dict.naver.com/#/entry/koen/8aa8386ae3144d2eaaef71a347ab7b75)  
Is there reason to wrote like this?

**Review by Creta5164 — CHANGES_REQUESTED** [review]
Most of them is nicely done, but there's something have problem.  
I wrote the mistranslated contents and the problematic parts here, so please correct them.

File: en/CommonEvents.json [hunk 9]
```diff
@@ -42,114 +42,114 @@
...
-        "퓨즈를 얻었다고?",
-        "좋은 소식이군.",
-        "\\!발전기만 찾으면 되겠네.",
-        "퓨즈를 찾은 곳과",
-        "같은 곳에 있지 않을까 싶은데.",
-        "상호작용을 원하는 사물 앞에서 ",
-        "메뉴의 아이템 창을 열어",
-        "아이템을 사용할 수 있습니다."
+        "You have a fuse?",
+        "Good news!",
```
Comment (Creta5164) [inline en/CommonEvents.json:68]: Someone suggested to change to 'That's good news.'.

File: en/CommonEvents.json [hunk 10]
```diff
@@ -42,114 +42,114 @@
...
+        "\\!Just need to find the generator.",
+        "Wonder if it's the same place",
```
Comment (Creta5164) [inline en/CommonEvents.json:70]: Someone suggested to change to 'Wonder if it's in the same place'

File: en/CommonEvents.json [hunk 11]
```diff
@@ -42,114 +42,114 @@
...
+        "Items can be used by opening",
+        "the item window from the menu",
+        "in front of the desired object."
     ],
     "514": [
-        "...그림자 못 봤냐고?",
-        "아까 전에 뭔가가 교무실 쪽으로",
-        "지나가긴 한 것 같은데.",
-        "한 번 찾아가 보는게 어때?"
+        "...Did you see the shadow?",
```
Comment (Creta5164) [inline en/CommonEvents.json:77]: Someone suggested to change to 'Did you see that shadow?'

File: en/CommonEvents.json (hunk 2 above)
Comment (Creta5164) [reply en/CommonEvents.json:145]: Someone suggested change to 'Some crazy guys must have kicked the computer because it was so slow'

Comment (Creta5164) [reply en/CommonEvents.json:145]: Also some more people suggested change to :
> Well, let's see.
> ...Huh, this wire was pulled again.
> Maybe some crazy guy kicked up this computer again to get faster.

**Review by Creta5164 — CHANGES_REQUESTED** [review]
Someone have reviewed more paragraphs, please check them.

File: en/CommonEvents.json (hunk 3 above)
Comment (BladeSides) [reply en/CommonEvents.json:153]: The but seems unnecessary, although I added it again.

**Comment by BladeSides** [comment]
Updated as requested.

**Comment by Creta5164** [comment]
I updated standalone demo, because I recently found some glitch happens in another language.  
So please download it and take to look them in-game!

I requested to reviewing this to another person about text only and in-game of this PR.  
Then we found silly things and bunch of outlandish situations while checking this.  

These common event content can found in `menu-phone call` and `use key items`.

**Comment by BladeSides** [comment]
Will add another pull request after improving quality of translation.

**Pull request closed without merging**